src-executables/Main-trident.hs

```diff
@@ -163,6 +163,10 @@ forgeOptParser = ForgeOptions <$> parseBasePaths
 genoconvertOptParser :: OP.Parser GenoconvertOptions
 genoconvertOptParser = GenoconvertOptions <$> parseBasePaths
                                           <*> parseOutGenotypeFormat
+                                          <*> parseRemoveOld
+
+parseRemoveOld :: OP.Parser Bool
+parseRemoveOld = OP.switch (OP.long "remove-old" <> OP.help "Remove the old genotype files when creating the new ones")
```

Comment:
Afaik all our option names are in camelCase at the moment - so maybe `removeOld` would fit in better than `remove-old`

Reply:
Ah ok. Fine. I agree